website/docs/en/guide/features/esm.mdx

```diff
@@ -0,0 +1,95 @@
+import { Stability } from '@components/ApiMeta';
+
+# ESM output
+
+Rspack supports bundling ESM format output. By default, Rspack generates CommonJS-like output, but you can configure `output.module` to generate ESM format output.
```

Comment:
```suggestion
Rspack supports bundling with ESM format output. By default, Rspack generates bundled output that includes Rspack runtime code rather than standard CommonJS or ESM format. You can configure [output.module](/config/output#outputmodule) to generate standard ESM format output.
```

Reply:
even if we use `outputModule`, the Rspack runtime will still be present. therefore, the Rspack runtime is not exclusive to CJS/ESM. what if we write "jsonp" directly?